definitions/infra-prometheus-redis/definition.yml

```diff
@@ -0,0 +1,22 @@
+domain: INFRA
+type: REDIS
+identifier: entity.name
+name: entity.name
+
+conditions:
+  - attribute: metricName
+    prefix: redis.
+
+tags:
+  - redis.aofLastBgrewriteStatus
+  - redis.aofLastWriteStatus
+  - redis.port
+  - redis.rdbLastBgsaveStatus
+  - redis.version
+
+goldenTags:
+  - redis.clusterRole
```

Comment:
Final comment from me I think, the `redis.clusterRole` also needs to be on the `tags` section. We should update this on the docs.

Reply:
Yeah I was a bit confused by the goldenTags section. Some more details in the docs would help.

Reply:
We'll use this experience to improve things. Thanks for the patience!

Reply:
Basically, the `tags` section includes all the tags that should be associated with the entity while the `goldenTags` are just a subset with the tags users consider most important.

Reply:
Unrelated to the pr but maybe we need to move golden tags outside of the syntethization.. but more files is more burden too :(